MedallionShell.Tests/Streams/MergedLinesEnumerableTestAsync.cs

```diff
@@ -0,0 +1,136 @@
+﻿#if NETCOREAPP2_2_OR_GREATER
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Medallion.Shell.Streams;
+using Moq;
+using NUnit.Framework;
+
+namespace Medallion.Shell.Tests.Streams
+{
+    public class MergedLinesEnumerableTestAsync
```

Comment:
- [ ] Should I keep this or change this to `public partial class MergedLinesEnumerableTest`?

Reply:
Given that, going forward, all test cases should be replicated to test both the async and sync variants, I’d prefer to have one file with interleaved cases. For example, TestOneIsEmpty and TestOneIsEmptyAsync should be right next to each other. That way if you find a need to change one it’s easy to remember to update the other.